core/src/main/resources/lib/form/dismiss.jelly

```diff
@@ -0,0 +1,48 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi, Yahoo! Inc.
```

Comment:
I'm not sure if we still need such license headers. But when we need then you should use the correct dates and authors.